dmitryi_bondar/task_3_OS_Linux/get_url.sh

```
@@ -0,0 +1,20 @@
+: << 'COMMENT'
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)